workshop/exercises/pygeoapi.config.yml

```diff
@@ -184,6 +184,37 @@ resources:
               data: tests/data/ne_110m_admin_0_countries.sqlite
               id_field: ogc_fid
               table: ne_110m_admin_0_countries
+    airports:
+        type: collection
+        title: airports of the world
```

Comment:
Are these the airports of the world or the airports of India?

Reply:
these are across the globe